test/screenshot/assertions/compareScreenshot.js

```diff
@@ -72,21 +82,25 @@ exports.assertion = function assertion(filename, baselinePath, browserName) {
   }
 
   this.pass = function passFn(value) {
-    const pass = value <= this.expected
+    let pass = value <= this.expected
+    
+    if (pass === false && some(['-u', '--updateScreenshot'], flag => process.argv.indexOf(flag) > -1)) {
```

Comment:
this won't scale very well, it's also fine for now but we probably want to parse all parameters first and then simply pass an `options` object